src/core/test/benchmarks/src/main/scala/Benchmarks.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import com.microsoft.ml.spark.FileUtilities.{File, readFile, writeFile}
+import org.apache.hadoop.fs.FileSystem
+
+import scala.collection.mutable.ArrayBuffer
+
+abstract class Benchmarks extends TestBase {
+  val moduleName: String
+  val targetDirectory = new File("target")
+  val thisDirectory = {
+    // intellij runs from a different directory
+    val d = new File("src/test/scala")
+    if (d.isDirectory) d else new File(s"$moduleName/src/test/scala")
+  }
+  assert(thisDirectory.isDirectory, "-- the test should run in the sub-project root level")
```

Comment:
Can we try to get rid of this dependency, it comes back to haunt us in fuzzing

Reply:
(I believe this is resolved - let me know if it still needs to be taken care of and we can discuss)

Reply:
Still seems to be in the file, could you remove it? Thanks!